.pipelines/templates/.builder-release-template.yaml

```diff
@@ -111,58 +111,114 @@ steps:
     retryCountOnTaskFailure: 3
     displayName: Building VHD
   - bash: |
+        echo MODE=$(MODE) && \
```

Comment:
this might be something you can look into removing, `MODE` should be redundant at this point since it's only used to differentiate between windows/linux (it previously differentiated between modes which haven't existed for a while...), which we can already do using `OS_TYPE`